uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -597,6 +599,34 @@ define([
       });
     }
 
+    /**
+     * Parses value depending on the specified type
+     *
+     * @param  {String} value
+     * @param  {String} type "boolean|decimal|integer"
+     * @return {Boolean|String}
+     */
+    function parseLoadedValue (value, type) {
+      var parsers = {
+        boolean: function (value) {
+          return value === '1';
+        },
+        decimal: function (value) {
+          return parseFloat(value).toString();
```

Comment:
This is a bit different than what you previously had. This will give you all the decimal points. Previously you required only the first one. Is this fine?

Reply:
Yes, I doublechecked that both FE and BE do not allow more than 1 after comma digit.